src/generated/ldb_chunks.h

```diff
@@ -1276,7 +1276,9 @@ namespace LDB_Reader {
 			/** String - RPG2003 */
 			frame_name				= 0x64,
 			/** Flag - RPG2003 */
-			invert_animations		= 0x65 
+			invert_animations		= 0x65,
+			/** When false the title is skipped and the game starts directly. In TestPlay mode skips directly to the Load scene. Added in RPG Maker 2003 v1.11 */
```

Comment:
This is not entirely true - it starts the load scene only when HideTitle is used (not just test play).